# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/samba-remove-lease.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- samba-3.0.26/source/modules/vfs_default.c.vanilla	2007-10-07 14:19:55.000000000 +0200
+++ samba-3.0.26/source/modules/vfs_default.c	2007-10-07 14:27:58.000000000 +0200
@@ -813,7 +813,7 @@
 
 	START_PROFILE(syscall_linux_setlease);
 
-#ifdef LINUX
+#if defined (LINUX) && ! defined (__UCLIBC__)
 	/* first set the signal handler */
 	if(linux_set_lease_sighandler(fd) == -1)
 		return -1;
